pkg/auth/data/authconfig_data.go

```diff
@@ -20,66 +20,66 @@ import (
 )
 
 func AuthConfigs(management *config.ManagementContext) error {
-	if err := addAuthConfig(github.Name, client.GithubConfigType, false, management); err != nil {
+	if err := addAuthConfig(github.Name, client.GithubConfigType, false, false, management); err != nil {
```

Comment:
Feels like this could be better expressed as two functions?

`addAuthConfig` and `addAuthConfigWithSLO` ?

Reply:
done. commit 3b5168e2824999c0f569ba78924028cf7a4af3d9